src/src/AdminBundle/Controller/EventController.php

```diff
@@ -84,6 +99,80 @@ public function deleteAction($id, $event_id, Request $request)
         $em->remove($event);
         $em->flush();
 
-        return $this->redirectToRoute('event_type_edit', ['id' => $id]);
+        return $this->redirectToRoute('event_edit', ['id' => $id, 'event_id' => $event_id]);
+    }
+
+    /**
+     * @Route("event_type/edit/{id}/event/{event_id}/edit/unassign/organizer/{organizer_id}", name="event_unassign_organizer")
+     * @Method("POST")
+     */
+    public function unassignOrganizerAction($id, $event_id, $organizer_id, Request $request)
+    {
+        $em = $this->getDoctrine()->getManager();
+
+        $eventOrganizer = $em->getRepository(EventOrganizers::class)->findOneBy(
+            array(
+                'eventId' => $event_id,
+                'organizerId' => $organizer_id,
+            )
+        );
+
+        if (!$eventOrganizer) {
+            throw $this->createNotFoundException(
+                'No organizer found for id '.$organizer_id.$event_id
+            );
+        }
+
+        $em->remove($eventOrganizer);
+        $em->flush();
+
+        return $this->redirectToRoute('event_edit', ['id' => $id, 'event_id' => $event_id]);
+    }
+
+    /**
+     * @Route("event_type/edit/{id}/event/{event_id}/edit/assign/organizer", name="event_assign_organizer")
+     * @Method("POST")
+     */
+    public function assignOrganizerAction($id, $event_id, Request $request)
+    {
+        $em = $this->getDoctrine()->getManager();
+        //todo zabezpecit, aby nenasiel deleted organizera
+        $organizer = $em->getRepository(Organizer::class)->findOneBy(['email' => $request->get('birds')]);
```

Comment:
co robi `$request->get('birds')` ? a co presne znamena `birds`?

Reply:
uz chapem co to robi no `birds` asi nieje spravny nazov.

Reply:
viem, copypaste 😀